sdcm/utils/alternator.py

```diff
@@ -8,6 +8,7 @@
 
 def create_table(endpoint_url, test_params):
     dynamodb_primarykey_type = test_params.get('dynamodb_primarykey_type', 'HASH')
+    write_isolation = test_params.get('alternator_write_isolation', 'f')  # default to forbid LWT
```

Comment:
don't we want the default to be ```'o' - use LWT only for requests that require read-before-write```

Reply:
good question... meanwhile the default is always... (maybe we should leave the default as default)

Reply:
changed the default

Reply:
so now, if ```alternator_write_isolation``` is empty, it will be the scylla default (meaning, always)?

Reply:
Yes